src/rpc/blockchain.cpp

```diff
@@ -2712,7 +2712,7 @@ UniValue CreateUTXOSnapshot(
     auto write_coins_to_file = [&](AutoFile& afile, const Txid& last_hash, const std::vector<std::pair<uint32_t, Coin>>& coins, size_t& written_coins_count) {
         afile << last_hash;
         WriteCompactSize(afile, coins.size());
-        for (auto [n, coin] : coins) {
+        for (const auto& [n, coin] : coins) {
```

Comment:
In commit b2cf3f77b14bccf11ad5bc211db7f746ae5073fb:
Nit: I think this should be moved to the first commit.

Reply:
done